awx/ui/client/src/shared/stateDefinitions.factory.js

```diff
@@ -584,27 +584,97 @@ function($injector, $stateExtender, $log, i18n) {
-                            template: `<add-rbac-resource users-dataset="$resolve.usersDataset" selected="allSelected" resource-data="$resolve.resourceData" without-team-permissions="true" title="` + i18n._('Add Users') + `"></add-rbac-resource>`
+                            template: `<add-rbac-resource default-params="$resolve.defaultParams" users-dataset="$resolve.usersDataset" selected="allSelected" resource-data="$resolve.resourceData" without-team-permissions="true" title="` + i18n._('Add Users') + `"></add-rbac-resource>`
                         }
                     },
                     ncyBreadcrumb:{
                         skip:true
                     },
                     resolve: {
-                        usersDataset: ['addPermissionsUsersList', 'QuerySet', '$stateParams', 'GetBasePath',
-                            function(list, qs, $stateParams, GetBasePath) {
+                        orgId:  ['$stateParams', 'Rest', 'GetBasePath', function($stateParams, Rest, GetBasePath) {
+                            let id;
+                            if ($stateParams.team_id) {
+                                Rest.setUrl(GetBasePath('teams') + `${$stateParams.team_id}`);
+                                id = Rest.get().then(({data}) => {
+                                    return data.summary_fields.organization.id;
+                                });
+                            } else {
+                                id = null;
+                            }
+                            return id;
+                        }],
+                        teamRoles:  ['$stateParams', 'Rest', 'GetBasePath', 'i18n', function($stateParams, Rest, GetBasePath, i18n) {
+                            let roles = null;
+                            if ($stateParams.team_id) {
+                                const basePath = GetBasePath('teams') + `${$stateParams.team_id}/object_roles`;
+                                Rest.setUrl(basePath);
+                                roles = Rest.get().then(({data}) => {
+                                    return data.results
+                                        .filter(({name}) => name === i18n._('Member') || name === i18n._('Admin'))
+                                        .map(({id}) => id)
+                                        .join(',');
+                                });
+                            }
+                            return roles;
+                        }],
+                        orgAdminRole:  ['$stateParams', 'orgId', 'Rest', 'GetBasePath', 'i18n',
+                            function($stateParams, orgId, Rest, GetBasePath, i18n) {
+                                let orgIdToCheck = $stateParams.organization_id || orgId;
+                                let role = null;
+                                if (orgIdToCheck) {
+                                    const basePath = GetBasePath('organizations') + `${orgIdToCheck}/object_roles`;
```

Comment:
This and the resolve in line 616 make a call to the same endpoint but are meant to filter out different role types ('Admin'  vs 'Member'). Is there a way to combine these two resolves? 

Otherwise there will be 2 extra API calls to `/organizations/${id}/object_roles/` when a user clicks on the "+" button on the add users view.

![Screen Shot 2019-03-18 at 1 38 05 PM](https://user-images.githubusercontent.com/2293210/54550834-2ada3e00-4983-11e9-8894-3e46af6e9dc0.png)